packages/core/src/TolgeeCore.ts

```diff
@@ -171,6 +171,11 @@ function createTolgee(options: TolgeeOptions) {
      */
     highlight: controller.highlight,
 
+    /**
+     * Find positions of keys in the DOM.
+     */
+    findPositions: controller.findPositions,
```

Comment:
This makes it "feature" update? Doesn't it?

Reply:
The API is changing regardless (with the UI package, although I'm not expecting it to influence anyone). So I'll make it feature update.